system/adb/Kconfig

```diff
@@ -0,0 +1,147 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig ADB_SERVER
+	bool "adb server command"
+	default n
+	---help---
+		Enable support for adb server.
+
+if ADB_SERVER
+
+config ADBD_STACKSIZE
+	int "adb stack size"
+	default 3072
+	---help---
+		The size of stack allocated for the adb task.
+
+config ADBD_PRIORITY
+	int "adb task priority"
+	default 100
+	---help---
+		The priority of the adb task.
+
+config ADBD_AUTHENTICATION
+	bool "adbd authentication support"
+	default n
+	---help---
+		Enable authentication for adb server.
+
+if ADBD_AUTHENTICATION
+
+config ADBD_AUTH_PUBKEY
+	bool "adbd public key authentication"
+	default n
+	---help---
+		Enable hook to accept new public keys.
+
+config ADBD_TOKEN_SIZE
+	int "authentication token size"
+	default 20
+
+endif # ADBD_AUTHENTICATION
+
+config ADBD_DEVICE_ID
+	string "default adb device id"
+	default ""
+
+config ADBD_PRODUCT_NAME
+	string "default adb product name"
+	default "adb dev"
+
+config ADBD_PRODUCT_MODEL
+	string "default adb product model"
+	default "adb board"
+
+config ADBD_PRODUCT_DEVICE
+	string "default adb product device"
+	default "NuttX device"
+
+config ADBD_FEATURES
```

Comment:
should we auto generate the feature string by check wether user select  ADBD_FILE_SERVICE, ADBD_SHELL_SERVICE...?